ui/Screen/Onboarding/Success.svelte

```diff
@@ -50,9 +61,16 @@
       You can find it at any time on your Profile or Settings page. You'll need
       to share it with others to collaborate.
     </p>
-
-    <Button dataCy="go-to-profile-button" on:click={next}>
-      Go to my projects
-    </Button>
+    <div class="buttons">
+      <Button
+        variant="transparent"
+        dataCy="go-to-profile-button"
+        on:click={profile}>
+        Go to profile
+      </Button>
+      <Button dataCy="go-to-wallet-button" on:click={wallet}>
+        Set up your wallet
+      </Button>
```

Comment:
The wallet is only enabled under the dev environment. Shouldn’t we guard it here, too?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1900/commits/b0e1ae3f4abdb9e210ec37314a80be6b9d95ce18